sentry/src/transport.rs

```diff
@@ -242,9 +252,10 @@ implement_http_transport! {
                                     disabled = Some(retry_after);
                                 }
                             }
+                            sentry_debug!("Got response: {:?}", resp.text());
```

Comment:
Two points here:
- The response must be consumed unconditionally, but at the moment it is conditional based on whether debug logs are emitted.
- If possible, can we avoid decoding to increase performance? You can still decode in the log, but not outside.